src/records/AssignmentCreationInterface.js

```diff
@@ -0,0 +1,6 @@
+import {Record, List} from 'immutable';
+
+export default Record({
+  openModal: false,
```

Comment:
This may change anyway based on the below, but I like to keep boolean properties/predicates prefixed with `is`, e.g. `isModalOpen` for this one.